From 3389b26f2ce509734b1e47fab5fef8930b3c4eb1 Mon Sep 17 00:00:00 2001
From: Thomas A Caswell <tcaswell@bnl.gov>
Date: Wed, 11 Jun 2025 23:14:47 -0400
Subject: [PATCH 1/2] MNT: restore missing __init__.py from tests module

This is required to run the tests on the installed package (which we do in the
conda-forge recipe)
---
 src/event_model/tests/__init__.py | 1 +
 src/event_model/tests/test_em.py  | 3 +++
 2 files changed, 4 insertions(+)
 create mode 100644 src/event_model/tests/__init__.py

diff --git a/src/event_model/tests/__init__.py b/src/event_model/tests/__init__.py
new file mode 100644
index 0000000..8b13789
--- /dev/null
+++ b/src/event_model/tests/__init__.py
@@ -0,0 +1 @@
+
diff --git a/src/event_model/tests/test_em.py b/src/event_model/tests/test_em.py
index 015072f..cfb892b 100644
--- a/src/event_model/tests/test_em.py
+++ b/src/event_model/tests/test_em.py
@@ -1021,3 +1021,6 @@ def test_resource_start_optional():
     event_model.compose_resource(
         spec="TEST", root="/", resource_path="", resource_kwargs={}
     )
+
+def test_import_tess():
+    import event_model.tests  # noqa: F401
-- 
2.50.1

